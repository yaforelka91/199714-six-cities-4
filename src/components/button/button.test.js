import React from 'react';
import renderer from 'react-test-renderer';
import Button from './button.jsx';

describe(`ButtonSnapshot`, () => {
  it(`should render Button`, () => {

    const tree = renderer.create(
        <Button>Button</Button>
    ).toJSON();

    expect(tree).toMatchSnapshot();
  });

  it(`should render disabled Button`, () => {

    const tree = renderer.create(
        <Button isDisabled={true}>Button</Button>
    ).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
