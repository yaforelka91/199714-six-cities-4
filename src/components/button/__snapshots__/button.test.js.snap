// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ButtonSnapshot should render Button 1`] = `
<button
  className="button"
  disabled={false}
  onClick={[Function]}
  type="button"
>
  Button
</button>
`;

exports[`ButtonSnapshot should render disabled Button 1`] = `
<button
  className="button"
  disabled={true}
  onClick={[Function]}
  type="button"
>
  Button
</button>
`;
